EventListener/ProjectListener.php

```diff
@@ -46,4 +47,16 @@ public function onTextmasterProjectInProgress(CallbackEvent $event)
         $project = $event->getSubject();
         $this->jobManager->startJobs($project);
     }
+
+    /**
+     * Launch project when translation memory is completed.
+     *
+     * @param CallbackEvent $event
+     */
+    public function onTextmasterProjectTmCompleted(CallbackEvent $event)
```

Comment:
If we stick to the other method it would be `onTextmasterProjectMemoryCompleted`.  
OK for you ?